src/components/item/item-reorder.scss

```diff
@@ -18,13 +18,18 @@ ion-reorder {
   font-size: 1.7em;
   opacity: .25;
 
-
   transition: transform 140ms ease-in;
 
   pointer-events: all;
   touch-action: manipulation;
 }
 
+.reorder-side-start ion-reorder {
+  @include transform(translate3d(-160%, 0, 0));
```

Comment:
in other to prevent DRY, could we use a scss variable like $reorder-initial-transform ?